lua errors

so when i spawn in acf and right click it like you would with a prop instead of a box showing up it gives me this error

[ERROR] addons/wire/lua/weapons/gmod_tool/stools/wire_debugger.lua:62: attempt to call field 'dummytrace' (a nil value)
  1. Filter - addons/wire/lua/weapons/gmod_tool/stools/wire_debugger.lua:62
   2. OpenEntityMenu - lua/includes/modules/properties.lua:81
    3. OnScreenClick - lua/includes/modules/properties.lua:125
     4. fn - lua/includes/modules/properties.lua:179
      5. Run - addons/ulib/lua/ulib/shared/hook.lua:109
       6. unknown - gamemodes/sandbox/gamemode/spawnmenu/contextmenu.lua:151

i need to fix it i run a gmod server but cant figure out how to fix it any help?

--- a/lua/wire/server/wirelib.lua
+++ b/lua/wire/server/wirelib.lua
@@ -994,26 +994,6 @@ function WireLib.GetOwner(ent)
 	return E2Lib.getOwner({}, ent)
 end
 
-function WireLib.dummytrace(ent)
-	local pos = ent:GetPos()
-	return {
-		FractionLeftSolid = 0,
-		HitNonWorld       = true,
-		Fraction          = 0,
-		Entity            = ent,
-		HitPos            = pos,
-		HitNormal         = Vector(0,0,0),
-		HitBox            = 0,
-		Normal            = Vector(1,0,0),
-		Hit               = true,
-		HitGroup          = 0,
-		MatType           = 0,
-		StartPos          = pos,
-		PhysicsBone       = 0,
-		WorldToLocal      = Vector(0,0,0),
-	}
-end
-
 function WireLib.NumModelSkins(model)
 	if NumModelSkins then
 		return NumModelSkins(model)

--- a/lua/wire/wireshared.lua
+++ b/lua/wire/wireshared.lua
@@ -144,6 +144,26 @@ end
 
 -- end extra table functions
 
+function WireLib.dummytrace(ent)
+	local pos = ent:GetPos()
+	return {
+		FractionLeftSolid = 0,
+		HitNonWorld       = true,
+		Fraction          = 0,
+		Entity            = ent,
+		HitPos            = pos,
+		HitNormal         = Vector(0,0,0),
+		HitBox            = 0,
+		Normal            = Vector(1,0,0),
+		Hit               = true,
+		HitGroup          = 0,
+		MatType           = 0,
+		StartPos          = pos,
+		PhysicsBone       = 0,
+		WorldToLocal      = Vector(0,0,0),
+	}
+end
+
 local table = table
 local pairs_sortvalues = pairs_sortvalues
 local ipairs_map = ipairs_map

